public-docs/creating-test-data.md

```diff
@@ -35,9 +40,14 @@ The `mockTag` output returns an object with fake data like this:
 }
 ```
 
+### Mock multiple objects
+
+Use the `makeMany` method and pass an integer to make multiple instances of one collection:
```

Comment:
"collection" is probably not quite the right word. Really it just mocks any object. Could be a document (top-level object of a collection) or any sub-object within a document, or any object you need anywhere as long as it has a schema that has been registered with `Factory` package.